br/pkg/backup/client.go

```diff
@@ -633,6 +633,12 @@ func BuildBackupSchemas(
 			default:
 				if tableInfo.SepAutoInc() {
 					globalAutoID, err = autoIDAccess.IncrementID(tableInfo.Version).Get()
+					// For a nonclustered table with auto_increment column, both auto_increment_id and _tidb_rowid are required.
+					// See also https://github.com/pingcap/tidb/issues/46093
+					if rowID, err1 := autoIDAccess.RowID().Get(); err1 == nil {
+						rowID = rowID + 1
```

Comment:
Can you explain why we need `rowID +=1` here and `*tbInfo.AutoIncIDExtra-1` in `ddl_api.go`(line 2633)?

Reply:
I follow the convention here https://github.com/pingcap/tidb/pull/46127/files/a6403bc4931d7ffa386cb06ea56f23c7dedb09fd#diff-e611f02c279107dbfa433418c809e99569c0b24c2ea92cc7aef9f7f41fbcc3b9L643